rcl_action/test/rcl_action/test_action_client.cpp

```diff
@@ -349,3 +351,56 @@ TEST_F(TestActionClientFixture, test_action_client_get_options) {
   options = rcl_action_client_get_options(&this->action_client);
   ASSERT_NE(options, nullptr) << rcl_get_error_string().str;
 }
+
+TEST_F(TestActionClientBaseFixture, test_action_client_init_fini_maybe_fail)
+{
+  RCUTILS_FAULT_INJECTION_TEST(
+  {
+    int64_t count = rcutils_fault_injection_get_count();
+    rcl_node_t node = rcl_get_zero_initialized_node();
+    rcl_node_options_t node_options = rcl_node_get_default_options();
+    std::string node_name = std::string("test_action_client_node_") + std::to_string(count);
+    rcl_ret_t ret = rcl_node_init(&node, node_name.c_str(), "", &this->context, &node_options);
```

Comment:
@brawner meta: should this be outside the loop?

We haven't agreed on this, but I think it'd be good to minimize the overlap between fault injection tests. It's an expensive test and the more stuff we add the harder it is to track it down. WDYT?

Reply:
Seems reasonable. Moved non rcl_action code outside of this macro loop.

Reply:
Uh, I don't see it here. Refreshing Github's UI won't do either :thinking:

Reply:
Hmm, phantom code change I see. Fixed again